### DISCUSSION TITLE
Welcome to blinko Discussions!

### DISCUSSION THREAD

**Discussion by blinko-space** [blinkospace/blinko#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by weado** [comment]
Hello, 
Do you have an Android app planned, as this tool is fantastic and replaces my Google Keep perfectly?
Thanks a lot !

**Reply by blessonism** [reply]
Also looking forward to the Android version.

**Reply by flyhunterl** [reply]
> Hello, do you have an Android app planned, as this tool is fantastic and replaces my Google Keep perfectly? Thanks a lot!

You can use PWA

**Comment by jjcfx** [comment]
关于Blinko的优化建议：

---

1、把闪念、笔记、待办，做成模块功能，可以再加一些功能，而且功能可增删，各模块可以相互自由穿插；

例如功能：闪念、笔记、待办、绘图、录音、拍照、脑图、大纲、PDF批注阅读、日历提醒、等等，可以慢慢开发；

每个功能模块在设置中可以选定，大家按需勾选。

全端左侧栏目可选用可排序，自由一点。

手机端：底部导航栏可自行选择，三个至五个，顺序等都能自由更换；在设置中选用和排序，不要在界面长按，容易误触；

---

2、安卓端的小部件现在只有录音，建议按模块每个都增加一个小部件；

例如在桌面可以轻松看到近期的待办和笔记等等；

待办要能直接在小部件勾选完成；

---

3、建议把设置按钮单独放出来，方便查找使用；

---

4、建议把标签整理单独放出来，现在的很繁琐；

---

5、所有客户端编辑内容，不支持空白行，这个不合理。

---

6、建议增强编辑栏，例如更改开源字体等；

---

7、新建待办时，建议自动添加《任务清单》按钮，不需要每次都要点一下；

---

8、建议待办增加期限，并申请系统通知权限，到时间会震动铃声提醒；

这样就能代替闹钟，代替日历；

---

9、建议增加文件夹功能，方便归档。现在的归档功能我不太理解，感觉用不上。

---

10、BUG：文档内部插入图片有显示BUG，无法显示；

---

11、BUG：安卓手机端的待办，没加清单发布后，双击进去再编辑，光标在最后时添加清单内容会消失

---

12、BUG：安卓手机端的待办，点击清单按钮后粘贴内容进去点发布，内容会消失

---

13、BUG：编辑内容时，用序号会自动被编号，而且去不掉

---

---
Optimization suggestions for Blinko:

---

1. Make flashes, notes, and to-dos into module functions. You can add some functions, and functions can be added or deleted, and each module can be freely interspersed with each other;

For example, functions: flash thoughts, notes, to-dos, drawing, recording, taking photos, mind maps, outlines, PDF annotation reading, calendar reminders, etc., can be developed slowly;

Each functional module can be selected in the settings, and you can check it as needed.

The left column of the full page can be sorted, which makes it more flexible.

Mobile version: The bottom navigation bar can be selected by yourself, from three to five, and the order can be freely changed; select and sort in the settings, do not long press on the interface, it is easy to accidentally touch;

---

2. The widgets on the Android side are currently only for recording. It is recommended to add a widget for each module;

For example, you can easily see recent to-dos and notes on the desktop;

To-do items must be checked and completed directly in the widget;

---

3. It is recommended to put the setting button separately for easy search and use;

---

4. It is recommended to sort out the labels and put them out separately. The current one is very cumbersome;

---

5. All client editing content does not support blank lines. This is unreasonable.

---

6. It is recommended to enhance the editing bar, such as changing open source fonts, etc.;

---

7. When creating a new to-do, it is recommended to automatically add the "Task List" button, so you don't need to click it every time;

---

8. It is recommended to add a deadline to the to-do list and apply for system notification permission. When the time comes, a ringtone will vibrate as a reminder;

This can replace the alarm clock and the calendar;

---

9. It is recommended to add a folder function to facilitate archiving. I don’t quite understand the current archiving function, and I don’t think it’s useful.

---

10. BUG: There is a display bug when inserting pictures into the document and cannot be displayed;

---

11. BUG: If you publish the to-do list on Android without adding a list, double-click it and then edit it, the cursor will disappear when you add the list content at the end.

---

12. BUG: In the to-do list on the Android mobile phone, if you click the list button and then paste the content into it and click Publish, the content will disappear.

---

13. BUG: When editing content, the serial number will be automatically numbered and cannot be removed.

---

**Comment by jjcfx** [comment]
I love blinko
